modules/core/src/main/java/org/apache/ignite/internal/cluster/DistributedTransactionConfiguration.java

```diff
@@ -0,0 +1,330 @@
+                        setDefaultValue(txOwnerDumpRequestsAllowed, dfltTxOwnerDumpRequestsAllowed, log);
+                    } else {
+                        longOperationsDumpTimeout.localUpdate(dfltLongOpsDumpTimeout);
+                        longTransactionTimeDumpThreshold.localUpdate(dfltLongTransactionTimeDumpThreshold);
+                        transactionTimeDumpSamplesCoefficient.localUpdate(dfltTransactionTimeDumpSamplesCoefficient);
+                        longTransactionTimeDumpSamplesPerSecondLimit.localUpdate(dfltLongTransactionTimeDumpSamplesPerSecondLimit);
+                        collisionsDumpInterval.localUpdate(dfltCollisionsDumpInterval);
+                        txOwnerDumpRequestsAllowed.localUpdate(dfltTxOwnerDumpRequestsAllowed);
+                    }
+                }
+            }
+        );
+    }
+
+    /**
+     * Cluster wide update of {@link #longOperationsDumpTimeout}.
+     *
+     * @param timeout New value of {@link #longOperationsDumpTimeout}.
+     * @return Future for {@link #longOperationsDumpTimeout} update operation.
+     * @throws IgniteCheckedException If failed during cluster wide update.
+     */
+    public GridFutureAdapter<?> updateLongOperationsDumpTimeoutAsync(long timeout) throws IgniteCheckedException {
+        return longOperationsDumpTimeout.propagateAsync(timeout);
+    }
+
+    /**
+     * Local update of {@link #longOperationsDumpTimeout}.
+     *
+     * @param timeout New value of {@link #longOperationsDumpTimeout}.
+     */
+    public void updateLongOperationsDumpTimeoutLocal(long timeout) {
+        longOperationsDumpTimeout.localUpdate(timeout);
+    }
+
+    /**
+     * @return Long operations dump timeout.
+     */
+    public Long longOperationsDumpTimeout() {
+        return longOperationsDumpTimeout.getOrDefault(dfltLongOpsDumpTimeout);
+    }
+
+    /**
+     * Cluster wide update of {@link #longTransactionTimeDumpThreshold}.
+     *
+     * @param longTransactionTimeDumpThreshold New value of threshold timeout in milliseconds.
+     * @return Future for {@link #longTransactionTimeDumpThreshold} update operation.
+     * @throws IgniteCheckedException If failed during cluster wide update.
+     */
+    public GridFutureAdapter<?> updateLongTransactionTimeDumpThresholdAsync(long longTransactionTimeDumpThreshold) throws IgniteCheckedException {
+        return this.longTransactionTimeDumpThreshold.propagateAsync(longTransactionTimeDumpThreshold);
+    }
+
+    /**
+     * Local update of {@link #longTransactionTimeDumpThreshold}.
+     *
+     * @param longTransactionTimeDumpThreshold New value of {@link #longTransactionTimeDumpThreshold}.
+     */
+    public void updateLongTransactionTimeDumpThresholdLocal(long longTransactionTimeDumpThreshold) {
+        this.longTransactionTimeDumpThreshold.localUpdate(longTransactionTimeDumpThreshold);
+    }
+
+    /**
+     * @return Threshold timeout for long transactions.
```

Comment:
Please add here and in other places corresponding links to full java doc like {@link DistributedTransactionConfiguration#longTransactionTimeDumpThreshold}